### ISSUE TITLE
Is support planned for 1.21.x?

### ISSUE THREAD

**Issue by BrainSlugs83** [Chocohead/OptiFabric#1958]
The latest versions I can find are for 1.20.x, and the documentation is still calling out stuff for like 1.16.x...

So... like the title is asking... is OptiFabric going to be supported for 1.21.x?

**Comment by Jetswing** [comment]
Yeah, wodering this too, i just dont get why the author could simply just make a small statement, and im by no means an expert on coding, but how hard can it REALLY be, it doesnt take SUCH a long time. If he could just make a statement.
